render/filters.go

```diff
@@ -244,6 +245,24 @@ func IsApplication(n report.Node) bool {
 // IsSystem checks if the node is a "system" node
 var IsSystem = Complement(IsApplication)
 
+// IsDesired checks if the node has the desired label
+func IsDesired(label string) FilterFunc {
+	return func(n report.Node) bool {
+		desiredKeyValue := strings.Split(label, "=")
```

Comment:
All this parsing should be done earlier, when parsing the flags. Scope should fail to start if the syntax is incorrect.